iceoryx_dds/include/iceoryx_dds/gateway/dds_to_iox.hpp

```diff
@@ -16,8 +16,9 @@
 #define IOX_DDS_GATEWAY_DDS_TO_IOX_HPP
 
 #include "iceoryx_dds/dds/dds_types.hpp"
-#include "iceoryx_dds/gateway/channel.hpp"
-#include "iceoryx_dds/gateway/dds_gateway_generic.hpp"
+#include "iceoryx_posh/popo/gateway/channel.hpp"
+#include "iceoryx_posh/popo/gateway/gateway_generic.hpp"
+#include "iceoryx_posh/popo/gateway/gateway_config.hpp"
```

Comment:
Great stuff! As `ìceoryx_posh` will be split up in the future, please move the generic gateway to a separate CMake target as discussed.